### ISSUE TITLE
Hubot install on OSX El Capitan

### ISSUE THREAD

**Issue by shigemk2** [github/generator-hubot#64]
I am trying to install the hubot on OSX El Capitan.

I completed this steps below.
```sh
$ npm install -g yo generator-hubot
$ mkdir hubot-example
$ cd hubot-example
```

I found this error when executing `yo hubot`.

```sh
$ yo hubot
                     _____________________________
                    /                             \
   //\              |      Extracting input for    |
  ////\    _____    |   self-replication process   |
 //////\  /_____\   \                             /
 ======= |[^_/\_]|   /----------------------------
  |   | _|___@@__|__
  +===+/  ///     \_\
   | |_\ /// HUBOT/\\
   |___/\//      /  \\
         \      /   +---+
          \____/    |   |
           | //|    +===+
            \//      |xx|

? Owner shigemk2 <xxxxxxxxxxxxxxxx>
? Bot name hubot-example
? Description A simple helpful robot for your Company
? Bot adapter (campfire) slackgot back false



/usr/local/lib/node_modules/yo/node_modules/rx/dist/rx.js:77
    throw e;
    ^
true
```

Please tell me the solution.

**Comment by alfee** [comment]
I faced same issue, I couldn't find a fundamental solution.
But I could success to avoid this issue; I pass all variables on command line.
e.g.
```
yo hubot --owner="your name <name@example.com>" --name="hubot-example" --description="A simple helpful robot for your Company" --adapter="slack"
```

**Comment by shigemk2** [comment]
Me too. I avoided the error by executing this command.

```sh
$ yo hubot --owner="your name <name@example.com>" --name="hubot-example" --description="A simple helpful robot for your Company" --adapter="slack"
```

**Comment by LoganBarnett** [comment]
Simply providing the adapter worked around the issue for me:
`yo hubot --adapter="slack"`

**Comment by yokomotod** [comment]
I found solution for this issue, but not sure why this occurred.

There're no changes around this code from long back, so it may be caused by yeoman(-generator) side change?

I'll check it and update PR description.

**Comment by yokomotod** [comment]
I think I could find the cause of this error and add a PR comment.

**Comment by scarletsky** [comment]
I have the same issue in ubuntu server 14.04.

**Comment by richardharrington** [comment]
I am experiencing this error as well. I have no idea why it is happening but I am sure it has to do with rx.js throwing an error, and NOT with that "got back false" message, which is a red herring that I think should probably be removed from the code. 

For an explanation, please see my [pull request](https://github.com/github/generator-hubot/pull/66)

**Comment by bstockus** [comment]
I have the same issue with Windows 10, Node v5.1.0

**Comment by mikechernev** [comment]
Apparently it's a yo issue and they had it before yeoman/yo#247
The _hack_ from @LoganBarnett worked for me too, so thanks 😉

**Comment by technicalpickles** [comment]
A fix for this released in `generator-hubot@0.4.0`. Anyone running into this should `npm install -g generator-hubot` to get the latest.

**Comment by shigemk2** [comment]
Thanks so much!!
